### PR TITLE
Add package-subscriptions-2019-11

### PR DESCRIPTION
**Pull request by jiasli**
1. Add `package-subscriptions-2019-11` in `readme.md` and `readme.python.md` to use 2019-11-01 API
2. Make corrections to previous PRs (see comments)

### CODE REVIEW

File: specification/resources/resource-manager/readme.md [hunk 1]
```diff
@@ -57,19 +57,10 @@ tag: package-managedapplications-2018-06
 ```
 
 ``` yaml $(package-deploymentscripts)
-tag: package-2019-11
+tag: package-deploymentscripts-2019-10-preview
```
Comment (jiasli) [inline specification/resources/resource-manager/readme.md]: Revert the change made in https://github.com/Azure/azure-rest-api-specs/pull/7839/files/8668e09bf93fd5a9cbaf00980a6cf50cdb6dee33#diff-d474de52c790d7028fb81d4abf0b9618R60, so that `$(package-deploymentscripts)` matches `tag: package-deploymentscripts-2019-10-preview`.

File: specification/resources/resource-manager/readme.md [hunk 2]
```diff
@@ -57,19 +57,10 @@ tag: package-managedapplications-2018-06
...
-### Tag: package-2019-11
-
-These settings apply only when `--tag=package-2019-11` is specified on the command line.
-
-```yaml $(tag) == 'package-2019-11'
-input-file:
-  - Microsoft.Resources/stable/2019-11-01/subscriptions.json
-```
-### Tag: package-resources-2019-10-preview
+### Tag: package-deploymentscripts-2019-10-preview
```
Comment (jiasli) [inline specification/resources/resource-manager/readme.md]: Fix a [mismatch](https://github.com/Azure/azure-rest-api-specs/pull/7486/files#diff-d474de52c790d7028fb81d4abf0b9618R63) made in https://github.com/Azure/azure-rest-api-specs/pull/7486. Let `package-deploymentscripts-2019-10-preview` in the title match that in the content.

File: specification/resources/resource-manager/readme.md [hunk 3]
```diff
@@ -358,6 +349,15 @@ input-file:
 - Microsoft.Resources/stable/2015-11-01/resources.json
 ```
 
+### Tag: package-subscriptions-2019-11
+
+These settings apply only when `--tag=package-subscriptions-2019-11` is specified on the command line.
+
+``` yaml $(tag) == 'package-subscriptions-2019-11'
+input-file:
+- Microsoft.Resources/stable/2019-11-01/subscriptions.json
```
Comment (jiasli) [inline specification/resources/resource-manager/readme.md]: Fix the [incorrect naming](https://github.com/Azure/azure-rest-api-specs/pull/7839/files/8668e09bf93fd5a9cbaf00980a6cf50cdb6dee33#diff-d474de52c790d7028fb81d4abf0b9618R64) of `package-2019-11` introduced in https://github.com/Azure/azure-rest-api-specs/pull/7839.

File: specification/resources/resource-manager/readme.md [hunk 4]
```diff
@@ -358,6 +349,15 @@ input-file:
...
+```
```
Comment (jiasli) [inline specification/resources/resource-manager/readme.md]: Fix the [incorrect naming](https://github.com/Azure/azure-rest-api-specs/pull/7839/files/8668e09bf93fd5a9cbaf00980a6cf50cdb6dee33#diff-d474de52c790d7028fb81d4abf0b9618R64) of `package-2019-11` introduced in https://github.com/Azure/azure-rest-api-specs/pull/7839.

**Review by yonzhan — APPROVED** [review]
LGTM

File: specification/resources/resource-manager/readme.md (hunk 2 above)
Comment (filizt) [reply specification/resources/resource-manager/readme.md]: This was intentional however making naming consistent in tags makes sense.

**Comment by filizt** [comment]
LGTM

**Review by Tiano2017 — APPROVED** [review]

**Review by erich-wang — APPROVED** [review]
